<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnEscribirArchivo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>escribir archivo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLeerArchivo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>310</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>leer archivo</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>31</y>
      <width>321</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEditarProducto">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>330</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>editar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAgregarProducto">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>agregar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEliminarProdcuto">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>330</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>eliminar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>26</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>76</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Web</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>126</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Direccion</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>176</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Telefono</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editNombreTienda">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="editWebTienda">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="editDireccionTienda">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="editTelefonoTienda">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelErrorNombreTienda">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelErrorWebTienda">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelErrorDireccionTienda">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelErrorTelefonoTienda">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>261</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ID - Nombre del producto, existencias</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionType_Here">
   <property name="text">
    <string>Type Here</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
